src/Libraries/Microsoft.Extensions.AI.Evaluation.Reporting/TypeScript/components/ConversationDetails.tsx

```diff
@@ -84,15 +84,16 @@ export const ConversationDetails = ({ messages, model, usage, selectedMetric }:
     const contextGroups = getContextGroups();
 
     return (
-        <div className={classes.section}>
+        <div className={classes.section} tabIndex={0} 
+                onKeyUp={e => e.key === 'Enter' && setIsExpanded(!isExpanded)}>
             <div className={classes.sectionHeader} onClick={() => setIsExpanded(!isExpanded)}>
                 {isExpanded ? <ChevronDown12Regular /> : <ChevronRight12Regular />}
                 <h3 className={classes.sectionHeaderText}>Conversation</h3>
                 {infoText && <div className={classes.hint}>{infoText}</div>}
             </div>
 
             {isExpanded && (
-                <div className={classes.sectionContainer}>
+                <div className={classes.sectionContainer}>  
```

Comment:
minor nit: trailing whitespace

```suggestion
                <div className={classes.sectionContainer}>
```